src/config/playpenconfig.rs

```diff
@@ -0,0 +1,68 @@
+use std::path::{PathBuf, Path};
+
+use super::tomlconfig::TomlPlaypenConfig;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct PlaypenConfig {
+    editor: PathBuf,
```

Comment:
Don't get me wrong. I think that this PR is really good, just there is no need to try to refactor everything at once now.

I was thinking about some possible future changes. Some random ideas:
- expanding the playpen config to make features opt in or opt out (clipboard support, resetable if editable, code runnable)
- split book.js by features or make it parametric like index hbs
- possibly support for alternative playpens like integer32 or other non rust speccific ones
- make mathjax opt in (it is really heavy and blocks offline reading now) 
- start working on alternative renderers (possibly split off the renderers into separate crates)
- make themes opt in or opt out (maybe some overridable list of defaults)